packages/app/pages/[modelName]/[documentTitle]/index.tsx

```diff
@@ -419,11 +475,18 @@ export async function getServerSideProps(ctx: NextPageContext) {
         decodeURIComponent(modelName.toString())
     )
 
+    // We only get the document collaborators here, because the browser's heuristics are needed to determine hasBeenActive and isActive.
+    const [collaboratorsError, collaborators] = await getDocumentCollaborators(
```

Comment:
We can load the page with a list of collaborators, but don't poll the API to set collaborators or get collaborators unless the user's browser has the `userActivation` API. I suppose this could be worked around by adding an `else` to the `userActivation` check, where we could still get collaborators on long poll.

Reply:
Addressed in 97c6f5c.